### PR TITLE
add partner tracker v2 docs

### PR DESCRIPTION
**Pull request by MarcoWorms**
broken link in md link checker requires merge to work

### CODE REVIEW

**Comment by MarcoWorms** [comment]
do not merge, this PR needs a rework, doing it today

File: docs/partners/integration_guide.md [hunk 1]
```diff
@@ -42,3 +49,4 @@ Implications:
 - Loss of TVL attributed if users transfer the vault tokens
```
Comment (DarkGhost7) [inline docs/partners/integration_guide.md:49]: Above it says partners deploy a routing contract for each vault then you say you can use multiple vaults per contract?

**Review by DarkGhost7 — APPROVED** [review]
lgtm other than my question

File: docs/partners/integration_guide.md (hunk 1 above)
Comment (MarcoWorms) [reply docs/partners/integration_guide.md:49]: well it's both according to docs: "A single routing contract can handle multiple vaults, but can also be deployed on a per vault basis"

**Pull request merged**